### PR TITLE
Update typescript.md

### PR DESCRIPTION
**Pull request by cmmer**
1、$ npm init egg 命令会生成 init 文件夹；
2、“在生产环境下我们建议将 ts 编译成 js 之后再运行” 有歧义，开发者会以为只是建议，不编译也行，应改成”在生产环境下我们**必须**将 ts 编译成 js 之后再运行“

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by MaledongGit** [comment]
@cmmer:

Thanks for your 1st contribution!

Several questions for you:

>$ npm init egg 命令会生成 init 文件夹；

Where's this? Are there any changes related to it in your submit?

>“在生产环境下我们建议将 ts 编译成 js 之后再运行” 有歧义，开发者会以为只是建议，不编译也行，应改成”在生产环境下我们必须将 ts 编译成 js 之后再运行“

I know you want to notify us that we CANNOT run ts files directly but just compile them into js first. However, what @whxaxes means is since egg can support ts files directly, so it can be run through ts-node as well with a low efficience.

Considering both of your suggestions, I think this can be changed something like:

egg-scripts 是用于在生产环境下运行 egg 的 cli ，考虑应用的健壮性和性能，因此我们不建议在线上环境使用 ts-node 直接运行应用，而是将 ts 文件编译成 js 后才运行。

**Review by MaledongGit — COMMENTED** [review]
@cmmer:

Thanks for your 1st contribution!

Several suggestions for you:

>$ npm init egg 命令会生成 init 文件夹；

Where's this? Are there any changes related to it in your submit?

>“在生产环境下我们建议将 ts 编译成 js 之后再运行” 有歧义，开发者会以为只是建议，不编译也行，应改成”在生产环境下我们必须将 ts 编译成 js 之后再运行“

I know you want to notify us that we CANNOT run ts files directly but just compile them into js first. However, what @whxaxes means is since egg can support ts files directly, so it can be run through ts-node as well with a low efficience, and in fact, ts-node will compile ts files into js files at background.

Considering both of your suggestions, I think this can be changed something like:

egg-scripts 是用于在生产环境下运行 egg 的 cli ，考虑应用的健壮性和性能，因此我们不建议在线上环境使用 ts-node 直接运行应用，而是将 ts 文件编译成 js 后才运行。

File: docs/source/zh-cn/tutorials/typescript.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ TypeScript 的静态类型检查，智能提示，IDE 友好性等特性，对
 通过骨架快速初始化：
 
 ```bash
-$ mkdir showcase && cd showcase
-$ npm init egg --type=ts
```
Comment (hubcarl) [inline docs/source/zh-cn/tutorials/typescript.md:30]: @cmmer  npm init  egg --type=ts 这种方式也我这边 Mac 系统是可以正常初始化项目的，`npm init egg 命令会生成 init 文件夹`  这个具体什么问题呢？

Comment (cmmer) [reply docs/source/zh-cn/tutorials/typescript.md:30]: 会在 showcase 文件夹下再自动生成 init 文件夹

Comment (cmmer) [reply docs/source/zh-cn/tutorials/typescript.md:30]: 而且还会出现 选择项目类型

File: docs/source/zh-cn/tutorials/typescript.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ TypeScript 的静态类型检查，智能提示，IDE 友好性等特性，对
...
+$ npm i -g egg-init
+$ egg-init showcase --type=ts
```
Comment (3imed-jaberi) [inline docs/source/zh-cn/tutorials/typescript.md:30]: I think this behave better;

- npx:

```bash
npx egg-init showcase --type=ts
```

- npm:

```bash
npm init egg showcase --type=ts
```

- yarn:

```bash
yarn create egg showcase --type=ts
```

**Pull request closed without merging**